src/main/java/com/danit/models/Client.java

```diff
@@ -52,6 +54,9 @@
   @Column(name = "email")
   private String email;
 
+  @OneToMany(mappedBy = "clientId", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
clientId or contractId?

Reply:
mappedBy means fk, so clientId is the right one